### PR TITLE
stop using tags to limit what we run

### PR DESCRIPTION
**Pull request by evgeni**
instead use two boolean vars to tell the build_package role whether it should wait and download logs.

also changes the default to wait, users can opt out by passing `-e build_package_wait=False`

Closes: #65

### CODE REVIEW

**Comment by evgeni** [comment]
@komidore64 would love your thoughts on this, as the original implementation was yours

**Comment by komidore64** [comment]
the main reason for the `always` tags was so that you had bunches of stuff that'd run without passing tags, but also run the same tasks **plus** tagged tasks, so i'm okay with this.

now since these playbooks are wrapped by obal, we're not running ansible directly, so i'm okay with them being removed. just make sure it works otherwise **i'm coming after you**.

**Comment by ehelms** [comment]
Could we consider making the wait functionality the default? Given most jobs need to wait.

**Comment by komidore64** [comment]
I support this.

**Comment by komidore64** [comment]
@evgeni 

make it so!

File: obal/data/roles/build_package/tasks/copr.yml [hunk 1]
```diff
@@ -49,25 +37,17 @@
...
       command: 'tito build --srpm --scl={{ scl }}'
       args:
         chdir: "{{ inventory_dir }}/packages/{{ inventory_hostname }}"
       register: srpm_build
-      tags:
-        - always
 
     - name: 'Run build'
-      command: "copr-cli build {{ copr_repo_name }} {{ srpm_build.stdout.split('Wrote: ')[1] }}"
+      command: "copr-cli build {% if not build_package_wait|bool %}--nowait{% endif %} {{ copr_repo_name }} {{ srpm_build.stdout.split('Wrote: ')[1] }}"
```
Comment (ehelms) [inline obal/data/roles/build_package/tasks/copr.yml]: I don't think I realized you could use this style of Jinja2 templating

**Comment by komidore64** [comment]
ack

**Pull request merged**